variation/main.py

```diff
@@ -541,13 +541,21 @@ async def hgvs_to_copy_number_change(
     return resp
 
 
-assembly_descr = "Assembly. If `accession` is set, will ignore `assembly` and `chr`. "\
-                 "If `accession` not set, must provide both `assembly` and `chr`."
+start0_descr = ("Start position (residue coords). If start is a definite range, this "
+                "will be the min start position")
+start1_descr = ("Only set when start is a definite range, this will be the max start "
+                "position")
+end0_descr = ("End position (residue coords). If end is a definite range, this will be "
+              "the min end position")
+end1_descr = ("Only set when end is a definite range, this will be the max end "
+              "position")
+start_pos_type = "Type of the start value in VRS Sequence Location"
+end_pos_type = "Type of the end value in VRS Sequence Location"
+assembly_descr = ("Assembly. If `accession` is set, will ignore `assembly` and `chr`. "
+                  "If `accession` not set, must provide both `assembly` and `chr`.")
 chr_descr = "Chromosome. Must set when `assembly` is set."
-accession_descr = "Accession. If `accession` is set, will ignore `assembly` and "\
-                  "`chr`. If `accession` not set, must provide both `assembly` and `chr`."  # noqa: E501
-start_descr = "Start position as residue coordinate"
-end_descr = "End position as residue coordinate"
+accession_descr = ("Accession. If `accession` is set, will ignore `assembly` and `chr`."
+                   " If `accession` not set, must provide both `assembly` and `chr`.")
```

Comment:
@korikuzma Maybe I'm reading this wrong -- but what happens if I set both `assembly` and `accession`?

Reply:
If `accession` is set, it will use that. Should I fix the wording?

Reply:
Maybe I should put that this is just for genomic atm.

Reply:
Oh, I'm sorry, I was misreading. I might suggest rephrasing to something like this for clarity

```
assembly_descr = ("Assembly. Ignored, along with `chr`, if `accession` is set. "
                  "If `accession` not set, must provide both `assembly` and `chr`.")
```